package.json

```diff
@@ -7,17 +7,20 @@
   "scripts": {
     "test": "./node_modules/.bin/mocha",
     "watch": "./node_modules/.bin/mocha -w",
-    "coverage": "istanbul cover ./node_modules/.bin/_mocha"
+    "coverage": "istanbul cover ./node_modules/.bin/_mocha",
+    "start": "./bin/probo-asset-receiver"
   },
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "aws-sdk": "^2.2.25",
```

Comment:
This is a production dependency not a dev dependency. Dev dependencies won't be installed on production deployments.